gettsim/social_insurance_contributions/beitr_bemess_grenzen.py

```diff
@@ -45,20 +39,19 @@ def _ges_krankenv_beitr_bemess_grenze_m(
     insurance contributions apply.
 
     """
-    out = wohnort_ost.replace(
-        {
-            True: soz_vers_beitr_params["beitr_bemess_grenze_m"]["ges_krankenv"]["ost"],
-            False: soz_vers_beitr_params["beitr_bemess_grenze_m"]["ges_krankenv"][
-                "west"
-            ],
-        }
-    )
-    return out.astype(float)
+    params = soz_vers_beitr_params["beitr_bemess_grenze_m"]["ges_krankenv"]
+
+    if wohnort_ost:
```

Comment:
```suggestion
    out = params["ost"] if wohnort_ost else params["west"]
```
is much more readable (that's why the construct was created fairly recently). No need to change everywhere, but this particular instance appears so often that it seems worth a search & replace to me.